src/plugins/monitoring/pom.xml

```diff
@@ -8,7 +8,7 @@
     </parent>
     <groupId>org.igniterealtime.openfire.plugins</groupId>
     <artifactId>monitoring</artifactId>
-    <version>1.7.0</version>
+    <version>1.7.0-SNAPSHOT</version>
```

Comment:
@GregDThomas did I do this right then?

Reply:
Yup :) 1.7.0 has yet to be released, so it should still be a SNAPSHOT